src/workflows/airtable/activities.ts

```diff
@@ -22,37 +22,48 @@ export interface ICreativesScoresAirtable extends FieldSet {
 }
 
 export interface ISoftrCreativesUser extends FieldSet {
-  Email: string;
+  Email?: string;
   "Token Address": string;
   "Wallet Address": string;
-  "Magic Link": string;
+  "Magic Link"?: string;
   "Start Date"?: string[];
-  Name: string;
+  Name?: string;
   Status: number;
   Level: number;
-  "Profile image"?: string[];
-  "Dashboard - Backend iFrame"?: string[];
 }
 
 export interface ICreativesMaster extends FieldSet {
   "Personal Email Address": string;
 }
 
+
 export async function findRecordWithEmail(
   email: string
 ): Promise<Record<ISoftrCreativesUser> | undefined> {
   console.log("find by email");
-  const base = (await getConnectionAirTable()).base("appxprwH6zsJbTFyM");
+  //const base = (await getConnectionAirTable()).base("appxprwH6zsJbTFyM");
+  const base = (await getConnectionAirTable()).base(process.env.SMART_CONTRACT_AIRTABLE_BASE!);
   /*const base = (await getConnectionAirTable()).base(
     `${process.env.CREATIVE_MASTER_AIRTABLE_BASE}`
   );*/
-  const findRecords = await base<ISoftrCreativesUser>("Users")
+  /*const findRecords = await base<ISoftrCreativesUser>("Users")
```

Comment:
let's not leave comments like this. These are not really helpful and mostly confusing

Reply:
This is done